src/pharmpy/model/external/nonmem/update.py

```diff
@@ -1055,6 +1055,8 @@ def match_advan4(odes):
     if len(outflows) != 1:
         return False
     central = outflows[0][0]
+    if central.name == "METABOLITE":
```

Comment:
There is a solution to this without using the compartment names in `match_advan2`. Perhaps this can be made into a more general function. For context: the problem is that we are matching an advan that could be used, but the reserved symbols in NONMEM are different so we would rather use the general linear advan.

Reply:
This has been implemented in the latest changes